### DISCUSSION TITLE
Ability to activate or deactivate a user via the API

### DISCUSSION THREAD

**Discussion by vlad-ko** [codecov/feedback#241] [Ideas]
**What product do you want to improve?**
Codecov API

**Is your feature request related to a problem? Please describe.**
Customer request

**Describe the solution you'd like**
Ability to "PATCH" a User record by sending something like username and maybe org. name to a new /user endpoint and additionally sending activate/deactivate switch as 1 or 0. 
The API should respond with the number of records successfully updated (ex: 1 user record updated successfully, or a return an error (ex: error updating user `<username>`. `<error message with details>`).

**Comment by rohan-at-sentry** [comment]
For visibility - this isn't currently on the roadmap. I'll convert this to a discussion so we can poll the broader community and gauge interest.
